best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
+* #it only expects one thing (within reason).
```

Comment:
Good point @kunzmann. If we were mocking everything but the unit then one expect per `it` would only be wasting half a dozen milliseconds... and the typing time.